packages/elements/vite.config.ts

```diff
@@ -31,6 +31,11 @@ export default defineConfig(
       },
       esbuild: esbuildOptions,
       build: {
+        emptyOutDir: !process.env.IIFE_BUILD,
+        rollupOptions: {
+          // For library usage, we want to externalize lit. When included as a script on the page we want to bundle it.
+          external: process.env.IIFE_BUILD ? undefined : [/^lit/],
```

Comment:
With this, lit _should_ be a dependency when using the root import, but I don't want to add lit as a dependency, for the use case where Stryker-js imports the packge and only needs the `mutation-test-elements.js` file. I'm not sure how this could be done. Maybe an `optionalDependencies`? @nicojs